securesystemslib/util.py

```diff
@@ -340,8 +361,9 @@ def import_json():
 
 json = import_json()
 
+json_str = Union[str, bytes, bytearray]
```

Comment:
I think we just need `Union[str, bytes]` here? https://github.com/python/typeshed/blob/master/stdlib/json/__init__.pyi#L36

That should be short enough that we can include it inline?

Reply:
[json's docs](https://docs.python.org/3/library/json.html#json.loads) said that but I agree on the shorter and practical annotation.